community/random-values/src/main/java/org/neo4j/values/storable/UTF8StringValueBuilder.java

```diff
@@ -29,7 +29,7 @@
  */
 class UTF8StringValueBuilder
 {
-    private static final int DEFAULT_SIZE = 8;
+    private static final int DEFAULT_SIZE = Byte.SIZE;
```

Comment:
This kind of implies that there is some connection between the size of a byte and the size of the backing array. The default size is more like the "typical ascii string length" - I just thought 16 was too big and 4 too small but there is no connection to the size of the individual bytes.

Reply:
in my impulse i though that was a default codepoint size of some sort, yes definitely it does not make any sense and will be restored

Reply:
👍